packages/circuit-ui/index.ts

```diff
@@ -169,6 +169,12 @@ export { default as Table } from './components/Table';
 export type {
   TableProps,
   TableSortDirection,
+  TableHeaderCell,
+  TableRowCell,
+  /**
+   * @deprecated
+   * Use `TableRowCell` or `TableHeaderCell` instead.
+   */
```

Comment:
I think the JSDoc comment is propagated from the type definition, isn't it?

Reply:
Maybe—for some reason it doesn't seem to be working (at least in my IDE) for types? I'll check. Otherwise it's anyway a good reminder for us to remove the `Cell` type in the next major

Reply:
Strange 🤔 I was hoping to avoid the duplication but whatever, this will be removed soon anyway 🤷‍♂️

Reply:
Theoretically it [should be working](https://www.typescriptlang.org/play?#code/PQKhFgCgAIWgBAJgUwA4CdkGMCGAXZRaAVwGdloADAORwFtlLoBLAO1IJ0SlmCjwCeqCgHV0Ae1YBzWg2gBeaK2J0ARsnQBufkIqyKijujZTtkKADNirLHmaTopHAIASzABSt6yAFzQxkjLeAJTQAN480FiSpOIANsgAdHHiUu4ARG4ANNDp0ADUSiFQAL5QQA). Removed the duplication 🙂

Reply:
Done in 9f74fe5